pl.art.lach.slawek.apps.v2blankbrowser.json

```diff
@@ -0,0 +1,36 @@
+{
+  "id": "pl.art.lach.slawek.apps.V2BlankBrowser",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.38",
+  "sdk": "org.gnome.Sdk",
+  "command": "V2BlankBrowser",
+  "clear": [ "*.o"],
```

Comment:
I don't think this does what you think it does.
Since `*.o` never get installed there is no point in cleaning them up. Regardless of what the build is (since there is currently nothing to build)

Reply:
and ignored this.